### ISSUE TITLE
SearchParameter creation makes aidbox queries to fail

### ISSUE THREAD

**Issue by yoliva** [Aidbox/Issues#65]
I'm having issues because I created a new `SearchParameter` for `RequestGroup` but after that, all my queries are failing. I had to rollback and remove the SearchParameter. We know that the search parameter isn't properly built but that shouldn't cause a general failure. This is the search parameter I created:

```
{
  "name": "target",
  "type": "reference",
  "module": "fhir-4.0.0",
  "expression": [
    [
      "action"
    ]
  ],
  "id": "RequestGroup.action",
  "resource": {
    "reference": "Entity/RequestGroup"
  },
  "resourceType": "SearchParameter"
}
```
Thanks

**Comment by niquola** [comment]
Thank you, I will check it out!

**Comment by niquola** [comment]
Should be fixed in 0.4.1-SNAPSHOT. Can you test it?
